src/components/Checkbox/Checkbox.vue

```diff
@@ -19,19 +19,21 @@
     <span
       style="content: '';"
       :class="[
-        'checkmark w-4 h-4 mr-1 rounded-sm border-solid border border-gray-100 -left-5 absolute top-1',
+        'checkmark w-4 h-4 mr-1 rounded-sm border-solid border border-gray-200 -left-5 absolute top-1',
         { 'bg-white-300': disabled },
-        { '!bg-gray-100': disabled && checked },
-        { 'border-error': error },
-        { 'border-primary-500 bg-primary-500': checked }
+        { '!bg-gray-200': disabled && checked },
+        { 'border-black bg-black': checked && !disabled }
       ]"
       data-testId="checkmark"
     />
     <!-- eslint-disable-next-line vue/no-v-html -->
-    <span v-html="label" />
+    <span
+      :class="['ml-1 type-small-500', disabled ? 'text-gray-400' : 'text-gray-800']"
+      v-html="label"
+    />
     <span
       v-if="required"
-      class="text-sm text-error"
+      class="text-sm text-red-500"
```

Comment:
question: so I'm guessing we won't be using `error` or `success` classes anymore, but specific colors? or am I wrong?

Reply:
yes, we will be using the colors as specified on figma/zeplin, that way there is only one correct answer
for example currently our error/warning/success states interchangeably use error/success or color classes so there is no consistency - hopefully we'll get to change that to color classes as directed by design